services/records/records/models.go

```diff
@@ -0,0 +1,30 @@
+package records
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+type Record struct {
```

Comment:
what about Comments, Errors, processingStatus, ... that is present in the PUT

Reply:
They are read-only. I've update them to be "computed"

Reply:
I still don't see Errors nor processingStatus, ... which are present in the response of the POST endpoint

Reply:
Oups ^^'
It's fix now.